### ISSUE TITLE
Load game (especially from in game) launches the ship automatically.

### ISSUE THREAD

**Issue by royfalk** [vegastrike/Vega-Strike-Engine-Source#878]
The game will automatically launch the ship in certain situations.
1. When loading a game from mission computer and not the main menu.
2. When docking at a random ship. (?)

Test 1st scenario
1. Start new campaign.
2. Save game.
3. Load game. 

2nd scenario
This is probably because the save game does not properly save random ships.

**Comment by evertvorster** [comment]
Yes, I find this especially true when you have docked with a base that is transiting the system. 

Since that base may simply not be there the next time you load the game, you are left in space. 
Not sure what to do about this. Do you travel with the ship? Should we save the transient bases so you have somewhere to be when you restore the save game?

**Comment by BenjamenMeyer** [comment]
We shouldn't have any transient bases that do not exist between game runs that one could save at.
If you land and can save; you should follow the entity you landed at and take off from there the next time the game is loaded.

**Comment by royfalk** [comment]
> We shouldn't have any transient bases that do not exist between game runs that one could save at. If you land and can save; you should follow the entity you landed at and take off from there the next time the game is loaded.

When you dock at a transiting ship (not base), the game should save this ship as a base of operations. I think it sometimes does. When you take off, you should take off from the same coordinates and into the same scenario (enemy ships) you left. For now.

Yes. We can consider sleeping at an inn and launching again at a different system. It's doable, but raises questions. If you attack a capital ship with a fighter and then run and dock at a merchant ship. Should you really be secure?

The simplest solution is to limit docking to system objects, like in WC. These can be capital ships but they would be treated as space stations and orbit something. That may solve the bug.

**Comment by evertvorster** [comment]
The simple solution unfortunately does not work, because sometimes there is a base transiting a system. 
"Base" in this context are shipyards, gas mines, anything with an "SS" in the name. You should be able to dock at these.

**Comment by BenjamenMeyer** [comment]
> The simple solution unfortunately does not work, because sometimes there is a base transiting a system. "Base" in this context are shipyards, gas mines, anything with an "SS" in the name. You should be able to dock at these.

Those would be `transient`, but rather transitioning through the system. I see `transient` as vessels/objects that might not be there from one run to the next - e.g a merchant vessel that is not part of the story line but randomly generated for a time period. These probably should not be able to be landed at.

Anything transitioning through the system should be officially part of the solar system and therefore exist from run to run and not ephemeral; in such case it should be okay to load wherever that object is in the system even if its not in the same place it was when the game wrote the saved file.
